python/src/main/python/drivers/graphicsfuzz_piglit_converter.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 The GraphicsFuzz Project Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+import sys
+from typing import List
+
+# Note: We define a 'shader job' as the JSON file of a GraphicsFuzz shader.
+# Each shader job has a corresponding shader file that has the same base name
+# as the shader job - for example, a shader job file named 'variant_005.json' will
+# have a corresponding shader file 'variant_005.frag' or 'variant_005.comp' in 
+# the same directory.
+
+# Piglit test section headers
+REQUIRE_HEADER = '[require]'
+VERTEX_HEADER = '[vertex shader passthrough]'
+FRAGMENT_HEADER = '[fragment shader]'
+TEST_HEADER = '[test]'
+
+# Draw command for piglit to draw the shader's output. Required in the test header.
+DRAW_COMMAND = 'draw rect -1 -1 2 2'
+
+# Strings used to specify the GL version to use in a piglit test's
+# [require] header.
+GL_VERSION_STRING = 'GL >= '
```

Comment:
Hmm actually, it does work most of the time. If you want to keep this, perhaps add a check for if the GLSL version is >= 330, or if it is GLSL ES then check if the version is >= 300 es. Then you can do the dividing trick. Otherwise, I would just not output any GL nor GLES requirements. I assume that these requirements help piglit decide whether the tests make sense to run on the current driver, but I am not sure. I assume `GLSL ES >= x` implies that GLES is required? Not sure though.

Reply:
Looking at shader_runner, I think it checks which versions of GLSL are supported by the driver, so just the GLSL (ES) requirements should be sufficient. No need for the GL or GLES requirements.

Reply:
Thanks, I've removed it.